docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -307,6 +311,26 @@ Refer to the [Integrating asset metadata into Dagster+ Insights](/dagster-plus/i
 | <PyObject object="MaterializeResult" /> | An object representing a successful materialization of an asset.                                                                         |
 | <PyObject object="MetadataValue" />     | Utility class to wrap metadata values passed into Dagster events, which allows them to be displayed in the Dagster UI and other tooling. |
 
+---
+
+## Reference: standard asset metadata entries
+
+The following is a set of standard asset metadata entries that can be included in the dictionaries that you pass to `metadata` attributes of <PyObject object="asset" decorator />, <PyObject object="MaterializeResult" />, etc. Many of these receive special treatment in Dagster's UI.
```

Comment:
What kind of special treatment in the UI?

I don't think we need to list or describe everything, but we could give an example and then add a link to the UI section on this page (`#viewing-asset-metadata-in-the-dagster-ui`) if that's applicable

Reply:
👍 added. Ideally we'd link [here](https://docs.dagster.io/concepts/webserver/ui#asset-details), but it looks like the screenshot there is outdated in a relevant way, so I think best not to for now.